loginFunctions.js

```diff
@@ -0,0 +1,103 @@
+const ERRORMESSAGE = {
+  wrongEmail: "잘못된 이메일 형식입니다",
+  wrongPassword: "비밀번호를 8자 이상 입력해주세요",
+  emailIsEmpty: "이메일을 입력해주세요",
+  passwordIsEmpty: "비밀번호를 입력해주세요",
+  nicknameIsEmpty: "닉네임을 입력해주세요",
+  passwordIsNotSame: "비밀번호가 일치하지 않습니다.",
+};
+
+const requireContent = (e) => {
+  console.log("requireContent called for:", e.target.id);
+  let content = e.target.value;
```

Comment:
❗️ 수정요청
재할당하는 경우가 아니라면 const로 선언해주세요~

```suggestion
  const content = e.target.value;
```